packages/fiori-elements-writer/src/data/templateAttributes.ts

```diff
@@ -88,7 +88,7 @@ const templateLibs: TemplateLibs = {
         },
         [TemplateType.FlexibleProgrammingModel]: {
             baseComponent: appComponentLibFioriElements,
-            ui5Libs: [...commonUi5Libs[OdataVersion.v4], 'sap.fe.templates']
+            ui5Libs: [...commonUi5Libs[OdataVersion.v4].filter((lib) => lib !== 'sap.fe.templates'), 'sap.fe.core']
```

Comment:
A little confusing. Can we instead remove `sap.fe.templates` from `commonUi5Libs`, if it is no longer "common".